### PR TITLE
Not all fields should not be mandatory in advanced settings.

### PR DESCRIPTION
**Pull request by smn**
And currently there's no error reporting.

### CODE REVIEW

**Comment by smn** [comment]
unicore_mission_control

Added form error feedback + styling

**Comment by smn** [comment]
unicore_mission_control

**Comment by smn** [comment]
@codiebeulaine ready for review.

File: requirements.txt [hunk 1]
```diff
@@ -17,3 +17,4 @@ Django-ostinato
 elastic-git
 unicore-cms
 uwsgi==2.0.8
+django-celery-email==1.0.4
```
Comment (smn) [inline requirements.txt:20]: This is a requirement but not specified in this projects requirements file.

**Comment by codiebeulaine** [comment]
:+1: not quite understanding what is happening with the front end html stuff though, error catching?

**Comment by smn** [comment]
Yeah, that stuff renders the errors in red like the screenshot I posted here: https://github.com/universalcore/unicore-mc/pull/54#issuecomment-68868657. All those errors weren't being displayed earlier resulting in a lot of confusion where the page reloaded and you couldn't tell if stuff had been saved properly or not because there was no feedback to go by.

**Pull request merged**